src/ui/events.js

```diff
@@ -192,7 +194,7 @@ export class MapTouchEvent extends Event {
  */
 export class MapWheelEvent extends Event {
     /**
-     * The event type.
+     * The event type (one of {@link Map.event:wheel}).
```

Comment:
I don't think the "one of..." construction works here since there's only one item instead of a list of items. Maybe `The event type ({@link Map.event:wheel}).` or `The {@link Map.event:wheel} event type.` or even just `{@link Map.event:wheel}` (since the section header says "type" already) ❓ 🤔

Reply:
Agree – fixing!

Reply:
Agree – fixing!